### PR TITLE
add a class and field for natural persons

### LINKED ISSUE #264: Add a field and class for natural persons (project and contract officials and roles)
The reactive disclosure section of the CoST IDS includes the following elements:

> **Project officials and roles**
> Name and position of the highest authority in the procuring entity where the budget for this project is allocated (project owner)

> **Contract officials and roles**
> Name and position of the public official in charge of administration of the contract(s).

## Proposal

Based on the approach suggested in https://github.com/open-contracting/standard/issues/883#issuecomment-722785450:

* add a `people` array to the `Organization` object
* define a `Person` object with the following properties:
  * `id`
  * `name`
  * `jobTitle`
 
Example:

```json
{
  "parties": [
    {
      "id": "1",
      "name": "Waka Kotahi NZ Transport Agency",
      "people": [
        {
          "id": "1",
          "name": "Sir Brian Roche",
          "jobTitle": "Chair"
        }
      ]
    }
  ]
}
```

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes #264

### CODE REVIEW

**Review by pindec — APPROVED** [review]
Looks good!

File: schema/project-level/project-schema.json [hunk 1]
```diff
@@ -1426,6 +1435,32 @@
...
       ]
+    },
+    "Person": {
+      "title": "Person",
+      "description": "A natural person.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "Identifier",
+          "description": "A local identifier for this person within the people array.",
```
Comment (jpmckinney) [inline schema/project-level/project-schema.json]: Will the Person object always be used in a people array? If not, then this should be made more generic.

File: schema/project-level/project-schema.json [hunk 2]
```diff
@@ -1426,6 +1435,32 @@
...
+          "type": "string",
+          "minLength": 1
+        },
+        "name": {
+          "title": "Name",
+          "description": "The full name of this person.",
```
Comment (jpmckinney) [inline schema/project-level/project-schema.json]: To match construction of `jobTitle`:

```suggestion
          "description": "The full name of the person.",
```

**Review by jpmckinney — COMMENTED** [review]
Just a couple comments. We could apply #223 to the new fields (minItems on the new array, minLength on all the new strings, minProperties on the new object), or we can wait.

File: schema/project-level/project-schema.json (hunk 1 above)
Comment (duncandewhurst) [reply schema/project-level/project-schema.json]: Good point, probably best to allow for reuse. Fixed in https://github.com/open-contracting/infrastructure/pull/279/commits/3397485730d766c0b694496c447cb2cff6de926f

**Comment by jpmckinney** [comment]
There's a conflict. Otherwise, please re-request review if ready for review again.

**Comment by duncandewhurst** [comment]
> There's a conflict. Otherwise, please re-request review if ready for review again.

Resolved!

> Just a couple comments. We could apply #223 to the new fields (minItems on the new array, minLength on all the new strings, minProperties on the new object), or we can wait.

Let's wait until all the schema changes for the reactive disclosures are done - I've removed #223 from the 1.0 milestone and added it to the reactive disclosures project so we don't forget.

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**